app/controllers/settings/apps/unsplash.js

```diff
@@ -9,15 +9,17 @@ export default Controller.extend({
     config: injectService(),
     unsplash: injectService(),
 
-    model: alias('settings.unsplash.firstObject'),
+    model: alias('settings.unsplash'),
     testRequestDisabled: empty('model.applicationId'),
 
     save: task(function* () {
         let unsplash = this.get('model');
         let settings = this.get('settings');
 
         try {
-            settings.get('unsplash').clear().pushObject(unsplash);
+            // TODO: Make validation work:
+            // yield unsplash.validate();
```

Comment:
@kevinansfield I tried to make the validation for the Unsplash app input and checkbox working, but failed. I commented this line out, which I thought should work, but it doesn't...